tachyon/zk/plonk/halo2/pcs_type.h

```diff
@@ -0,0 +1,48 @@
+#ifndef TACHYON_ZK_PLONK_HALO2_PCS_TYPE_H_
+#define TACHYON_ZK_PLONK_HALO2_PCS_TYPE_H_
+
+#include <stdint.h>
+
+#include <string>
+
+#include "absl/strings/substitute.h"
+
+#include "tachyon/base/flag/flag_value_traits.h"
+
+namespace tachyon {
+namespace zk::plonk::halo2 {
+
+// THE ORDER OF ELEMENTS ARE IMPORTANT!! DO NOT CHANGE!
+// This order matches with the constants of c APIs.
+// Pleas refer to tachyon/c/zk/plonk/halo2/constants.h for details.
+enum class PCSType : uint8_t {
+  kGWC,
+  kSHPlonk,
+};
```

Comment:
how about giving specific number to each element?

Reply:
why?